cmd/kube-controller-manager/app/certificates.go

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
> Or even "groups" as in api-groups?

I wasn't trying to change up packages yet. I'm not even completely sure that I want to.  I was just trying to slim down the main type and get some consistency for startup.